source/delegate/test/Delegate.js

```diff
@@ -474,18 +467,16 @@ contract('Delegate Integration Tests', async accounts => {
       })
 
       // authorize the delegate to be sender
-      let expiry = await getTimestampPlusDays(0.5)
-      let tx = await swapContract.authorizeSender(
+      /*      let tx = await swapContract.authorizeSender(
         aliceDelegate.address,
-        expiry,
         {
           from: aliceTradeWallet,
         }
       )
-      emitted(tx, 'AuthorizeSender')
+      emitted(tx, 'AuthorizeSender')*/
 
       // increase time past expiry
-      await advanceTime(SECONDS_IN_DAY * 0.6)
+      //await advanceTime(SECONDS_IN_DAY * 0.6)
```

Comment:
Why is this commented out - and the lines above? Can we not remove it?

Reply:
removing my bad. lmk if you find any others, i'll take a look as well.